server/models/account/user.ts

```diff
@@ -106,12 +123,66 @@ enum ScopeNames {
             }
           }
         ]
-      },
-      {
-        model: UserNotificationSettingModel,
-        required: true
       }
     ]
+  },
+  [ScopeNames.WITH_STATS]: {
+    attributes: {
+      include: [
+        literalVideoQuotaUsed,
+        [
+          literal(
+            '(' +
+              'SELECT COUNT("video"."id") ' +
+              'FROM "video" ' +
+              'INNER JOIN "videoChannel" ON "videoChannel"."id" = "video"."channelId" ' +
+              'INNER JOIN "account" ON "account"."id" = "videoChannel"."accountId" ' +
+              'WHERE "account"."actorId" = "UserModel"."id"' +
+            ')'
+          ),
+          'videoCount'
+        ],
+        [
+          literal(
+            '(' +
+              'SELECT ("abuses", "acceptedAbuses")::text ' +
+              'FROM (' +
+                'SELECT COUNT("videoAbuse"."id") AS "abuses", ' +
+                       `COUNT(CASE WHEN "videoAbuse"."state" = ${VideoAbuseState.ACCEPTED} THEN 1 END) AS "acceptedAbuses" ` +
```

Comment:
Try to use `FILTER`: https://medium.com/little-programming-joys/the-filter-clause-in-postgres-9-4-3dd327d3c852